snpdb/variant_pk_lookup.py

```diff
@@ -216,8 +216,8 @@ def __init__(self, *args, **kwargs):
     def _get_locus_hash(self, contig_id, position, ref_id):
         return contig_id, position, ref_id
 
-    def _get_variant_hash(self, contig_id, position, ref_id, alt_id):
-        return contig_id, position, ref_id, alt_id
+    def _get_variant_hash(self, contig_id, start, end, ref_id, alt_id):
+        return contig_id, start, end, ref_id, alt_id
```

Comment:
Not a big deal, but should this be the default implementation that you can override? It's straight forward enough.

Reply:
The fact it's in a class hierarchy is vestigial as I had parallel implementations before removing the other one. Will probably collapse it to 1 concrete class